README.md

```diff
@@ -3,9 +3,9 @@
   <h3 align="center">dicom</h3>
   <p align="center">High Performance Golang DICOM Medical Image Parser<p>
   <p align="center"> 
-    <a href="https://github.com/suyashkumar/dicom/actions"><img src="https://github.com/suyashkumar/dicom/workflows/build/badge.svg" /></a> 
-    <a href="https://godoc.org/github.com/suyashkumar/dicom"><img src="https://godoc.org/github.com/suyashkumar/dicom?status.svg" alt="" /></a>
-    <a href="https://goreportcard.com/report/github.com/suyashkumar/dicom"><img src="https://goreportcard.com/badge/github.com/suyashkumar/dicom" alt=""></a> 
+    <a href="https://github.com/amitbet/dicom/actions"><img src="https://github.com/amitbet/dicom/workflows/build/badge.svg" /></a> 
```

Comment:
can you please restore the links in the README to this repo, including the imports so that the automated tests can run? Thanks!